src/components/menu/menu.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import styles from "./menu.module.scss";
+
+export const Menu = () => {
+  const menu = ["all", "documentary", "comedy", "horror", "crime"];
+  const sortOptions = ["Release date", "Title"];
+
+  const [selectedMenu, setSelectedMenu] = useState("all");
+  
+  return (
+    <div className={styles.menuWrapper}>
+      <nav className={styles.menuItems}>
+        {menu.map((item) => {
+            const active = item === selectedMenu
+            return (
+                <div key={item} className={`${styles.menuItem} ${active ? styles.active : ''}`} onClick={(e) => setSelectedMenu(item)}>{item}</div>
```

Comment:
`e` arg is not used anywhere, so, I advise to always avoid unused code parts/variables, they always pollute code.